File: src/main/resources/template/android/AndroidLauncher.java
package ${package};

import android.os.Bundle;
import com.jme3.app.AndroidHarness;

public class AndroidLauncher extends AndroidHarness {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        app=new Main();
        super.onCreate(savedInstanceState);

    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: src/main/resources/template/android/AndroidLauncher.kt
package ${package}

import android.os.Bundle
import com.jme3.app.AndroidHarness

class AndroidLauncher : AndroidHarness() {
    fun onCreate(savedInstanceState: Bundle) {
        app = Main()
        super.onCreate(savedInstanceState)

    }

    override fun onDestroy() {
        super.onDestroy()
    }
}